jose/jwk.py

```diff
@@ -18,6 +17,11 @@
 from jose.exceptions import JWSError
 from jose.exceptions import JOSEError
 
+if hasattr(RSA, '_RSAobj'):
```

Comment:
PyCryptodome's RSA module doesn't have the [_RSAobj class](https://www.dlitz.net/software/pycrypto/api/current/Crypto.PublicKey.RSA._RSAobj-class.html); Instead, it has a [RsaKey class](http://legrandin.github.io/pycryptodome/Doc/3.1/Crypto.PublicKey.RSA.RsaKey-class.html) which is almost the same thing.